src/setup/constants.rs

```diff
@@ -17,3 +19,15 @@ pub const PRIVATE_NETWORK_DIR: &str = "private_network";
 
 /// Node directory without an index. The correctly indexed node directory is "Node0".
 pub const NODE_DIR: &str = "Node";
+
+/// The address on which the relay node listens for incoming connections.
+///
+/// Non-relay nodes do not have this address configured.
+/// The address is named `NetAddress` in the official Algorand documentation.
+pub const NET_ADDR_FILE: &str = "algod-listen.net";
+
+/// The address on which the node listens for REST API calls.
+pub const REST_ADDR_FILE: &str = "algod.net";
```

Comment:
Well, we could add the official name here as well.

Reply:
yeah, if it's just the REST-related stuff, the name should be more descriptive than just `algod`